toltec/builder.py

```diff
@@ -316,13 +316,15 @@ def _build(self, recipe: Recipe, src_dir: str) -> None:
             )
 
         if host_deps:
-            opkg_conf_path = "$SYSROOT/etc/opkg/opkg.conf"
+            opkg_conf_path = "$SYSROOT_AARCH64/etc/opkg/opkg.conf" if recipe.arch == "rmpp" else "$SYSROOT/etc/opkg/opkg.conf"
+            opkg_exec = "opkg-aarch64" if recipe.arch == "rmpp" else "opkg"
+
             pre_script.extend(
                 (
                     'echo -n "dest root /',
                     "arch all 100",
-                    "arch armv7-3.2 160",
-                    "src/gz entware https://bin.entware.net/armv7sf-k3.2",
+                    f"arch {'aarch64-3.10' if recipe.arch == 'rmpp' else 'armv7-3.2'} 160",
```

Comment:
This will unfortunately not work for when we do os level splits in the arch: https://github.com/toltec-dev/toltec/blob/stable/docs/package.md#architecture-section

We will be having `rmpp` and `rmppos3` initially, but eventually there may be a `rmppos4` or some other breakpoint.

Reply:
Fixed!